x-pack/plugins/enterprise_search/common/ml_inference_pipeline/index.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { MlTrainedModelConfig } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+
+import { MlInferencePipeline } from '../types/pipelines';
+
+// Getting an error importing this from @kbn/ml-plugin/common/constants/data_frame_analytics'
+// So defining it locally for now with a test to make sure it matches.
+export const BUILT_IN_MODEL_TAG = 'prepackaged';
```

Comment:
This is the error I was getting trying to import this from the ML plugin instead:

```
       │          ERROR in ./common/ml_inference_pipeline/index.ts
       │          Module not found: Error: import [../../../ml/common/constants/data_frame_analytics] references a non-public export of the [ml] bundle and must point to one of the public directories: [public, common]
       │           @ ./common/ml_inference_pipeline/index.ts 7:0-87 64:142-160 65:105-123
       │           @ ./public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/ml_inference_logic.ts
       │           @ ./public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/add_ml_inference_pipeline_modal.tsx
       │           @ ./public/applications/enterprise_search_content/components/search_index/pipelines/pipelines.tsx
       │           @ ./public/applications/enterprise_search_content/components/search_index/search_index.tsx
       │           @ ./public/applications/enterprise_search_content/components/search_index/search_index_router.tsx
       │           @ ./public/applications/enterprise_search_content/components/search_indices/search_indices_router.tsx
       │           @ ./public/applications/enterprise_search_content/components/search_indices/index.ts
       │           @ ./public/applications/enterprise_search_content/index.tsx
       │           @ ./public/plugin.ts
       │           @ ./public/index.ts
       │           @ /Users/rodney/Code/elastic/kibana/node_modules/@kbn/optimizer/target_node/src/worker/entry_point_creator.js
```

Reply:
Yeah, what you did here seems like the best way to deal with at this time.